src/qt/window.cpp

```diff
@@ -1737,7 +1724,11 @@ bool wxWindowQt::QtHandleMouseEvent ( QWidget *handler, QMouseEvent *event )
 
     // Use screen position as the event might originate from a different
     // Qt window than this one.
+#if QT_VERSION_MAJOR >= 6
+    const wxPoint mousePos = ScreenToClient(wxQtConvertPoint(event->globalPosition().toPoint()));
+#else
```

Comment:
It looks like this is done in a few places, perhaps it could be worth adding a function hiding the difference between `globalPos()` and `globalPosition()` and just call it from here and elsewhere?

Reply:
I added `wxQtGetEventPosition()` to hide mouse position retrieval in c12363e. Regarding `globalPosition()`/`globalPos()`, there are only two: this one and the other `globalPosition()`/`screenPos()` in `winevent.h`, which I left unchanged.